src/components/CompareResults/Retrigger/RetriggerButton.tsx

```diff
@@ -163,6 +180,9 @@ function RetriggerButton({ result }: RetriggerButtonProps) {
         onClick={() => void onRetriggerButtonClick()}
       >
         <RefreshOutlinedIcon />
+        {baseParentSignature || newParentSignature ? (
+          <span className={styles.retrigger}>Retrigger test</span>
+        ) : null}
```

Comment:
This doesn't look right (eg the hover style), we really need a normal button (variant "text" I believe) with a label in this case, such as https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

Therefore I'd create 2 simple components for just the buttons, taking "onClick" as a prop. You can create them in this file. `RetriggerIconButton` and `RetriggerFullButton` for example.
Then `RetriggerButton` would take the component to use as a prop `RetriggerButtonComponent`.
Finally each user of the `RetriggerButton` component could specify a different one.

Reply:
I thought about this some more and I think that part of my suggestion above is a bit overkill for our case here it's used only twice.
So here is my other simpler suggestion:
* make RetriggerButton take a `variant` prop taking 2 values `"icon" | "text"`
* use a ternary on this prop to decide whether to add the icon button or the text button